src/schema/BitmapSchema.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Define schema for a bitmap.
+ * @copyright Project CHIP Authors 2022
+ * @license Apache-2.0
+ */
```

Comment:
What do you think of using those short license / copyrights declarations?
They follow the JsDoc 3 from here: https://jsdoc.app/

Reply:
This is the compliant license style:
```
/**
 * @license
 * Copyright 2000 Example, Inc
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Compliant with Matter license declaration?

Per JsDoc 3 doc, the SPDX Id can be used directly after @license:
https://jsdoc.app/tags-license.html

For the @copyright tag, it is supposed to use in combination with a @file tag:
https://jsdoc.app/tags-copyright.html
Maybe should be @copyright 2022 Project CHIP Authors to follow the recommendation from inside Apache 2.0 license
https://github.com/project-chip/matter.js/blob/main/LICENSE#L189

I don't mind using your suggestion above or the full license declaration, I just thought the short version was neat...

Reply:
Yes, I like the proposed format too, but for now it is most expedient to stay with the format that has been pre-approved by legal.

Reply:
Ok, I will update that when I get a break in my day time job! (In 3h...)

Reply:
Updated